lib/internal/options.js

```diff
@@ -65,7 +65,7 @@ function generateConfigJsonSchema() {
   const schema = {
     __proto__: null,
     $schema: 'https://json-schema.org/draft/2020-12/schema',
-    additionalProperties: false,
+    additionalProperties: true,
```

Comment:
How come this is changing to be more permissive?

Is that because the entire package.json is validated instead of a specific property?